thirdparty/README.md

```diff
@@ -724,6 +724,23 @@ with the provided patch.
   unnecessary code.
 
 
+## oboe
+
+- Upstream: https://github.com/google/oboe
+- Version: 1.8.2 (16d72c89be9eb8a7d617a7be531a31dba3db74f1, 2024)
```

Comment:
Put version 1.8.2, although it's not released yet, just to have this fix: [google/oboe#1968](https://github.com/google/oboe/pull/1968). Without this fix, on some devices, the audio would be silent or with a lot of glitches since the code checking for it was never called